app/com/gu/contentapi/sanity/package.scala

```diff
@@ -23,10 +23,11 @@ package object sanity extends ScalaFutures with Matchers with IntegrationPatienc
   def incrementIncidentCount = {
     incidentCount += 1
     lastIncidentDateTime = Some(DateTime.now)
+    println(s"Incident count: $incidentCount")
   }
   def resetCount: Unit ={
```

Comment:
Small stylistic point: this method has a side-effect (it changes mutable state), so it should have parentheses.